packages/atlas-experiment-table/src/TableContent.js

```diff
@@ -1,141 +1,125 @@
-import TooltipIcon from './TooltipIcon'
+// We’re not using Link because Evergreen restyles the links and we want EBI VF for that
+import { Table, majorScale } from 'evergreen-ui'
+import { tableHeaderPropTypes } from './filterPropTypes'
 
-const fetchJson = async (endpoint, host, queryParams) => {
-  const url = URI(endpoint, host).search(queryParams).toString()
-  const response = await fetch(url)
-  return await response.json()
-}
-
-const alertInvalidFiles = async (host, checkedRows) => {
-  const endpoint = `json/experiments/download/zip/check`
-  try {
-    const response = await fetchJson(endpoint, host, {accession: checkedRows})
-    const data = response.invalidFiles
-    const invalidFiles = !_.isEmpty(data) && Object.keys(data).map((experiment) => `${data[experiment].join(`\n`)}`)
-    const downloadUrl = URI(`experiments/download/zip`, host).search({accession: checkedRows}).toString()
-    if (_.isEmpty(Object.values(data)[0])) {
-      window.location.replace(downloadUrl)
-    } else if (window.confirm(`The following files are not available.\n${invalidFiles.join(`\n`)}\nWould you like to continue?`)) {
-      window.location.replace(downloadUrl)
-    }
-  } catch (e) {
-    console.log(`error`, e)
-  }
-}
+import TableHeaderCell from './head/TableHeaderCell'
+import SelectionTableHeaderCell from './head/SelectionTableHeaderCell'
+import { _validateAndDownloadExperimentFiles } from './head/downloadHelper'
 
-const TableContent = ({tableHeader, searchedColumnIndex, searchQuery, orderedColumnIndex,
-  ascendingOrder, enableDownload, checkedRows, currentPageData, host,
-  tableHeaderOnClick, tableHeaderOnChange, downloadOnChange, downloadTooltip}) =>
-  <div className={`row expanded`}>
-    <div className={`small-12 columns`} >
-      <Table border>
-        <Table.Head>
+import TableCell from './TableCell'
+import SelectionTableCell from './SelectionTableCell'
 
-          {
-            tableHeaderCells(tableHeader, searchedColumnIndex, searchQuery, orderedColumnIndex, ascendingOrder,
-              columnNumber => tableHeaderOnClick(columnNumber),
-              (value, columnNumber) => tableHeaderOnChange(value, columnNumber)
-            )
-          }
+// Minimum width of the large size in EBI VF
+const ebiVfLargeMinWidth = 1024
 
-          {
-            enableDownload && <Table.TextHeaderCell className={`downloadHeader`} flexBasis={100} flexShrink={100} flexGrow={100}>
-              {
-                <div>
-                  {
-                    checkedRows.length > 0 ?
+const TableContent =
+  ({
+    // Presentational props
+    dataRows, tableHeaders,
+    // Filter handling
+    filters, tableHeaderCellOnChange,
+    // Ordering
+    sortColumnIndex, ascendingOrder, tableHeaderCellOnClick,
+    // Row selection and action on selected rows
+    rowSelectionColumn, selectedRows, host, selectOnChange
+  }) =>
+    <div className={`row expanded`}>
+      <div className={`small-12 columns`}>
```

Comment:
I think to keep the code as general as possible, we could pass the div `className` provided by EBI using props as what we did for some other components. And we may use the default props to make the input props simpler. 

Here are a bunch of `className` in `TableFooter` and `TablePreamble` too.

Reply:
I didn’t think of this and I think there’s value in being able to change the `className` attribute via props. Especially the outer one, since we want to have the table in the Cambridge Cell Atlas website that has a regular `row` (not `expanded`) layout.